configure.ac

```diff
@@ -1266,6 +1266,36 @@ AC_SUBST(OPENTHREAD_ENABLE_SERVICE)
 AM_CONDITIONAL([OPENTHREAD_ENABLE_SERVICE], [test "${enable_service}" = "yes"])
 AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_SERVICE],[${OPENTHREAD_ENABLE_SERVICE}],[Define to 1 if you want to enable Service])
 
+#
+# Long routes
+#
+
+AC_ARG_ENABLE(long_routes,
```

Comment:
I don't think the configure option here is necessary.   We could add a definition directly in [src/core/openthread-core-default-config.h](https://github.com/openthread/openthread/blob/master/src/core/openthread-core-default-config.h)

Reply:
Ok, I will remove it.